### ISSUE TITLE
Hemostat surgery step cannot be done

### ISSUE THREAD

**Issue by Mark9013100** [PolarisSS13/Polaris#363]
When attempting to use the hemostat, nothing will come up, and the step will have to be skipped.

**Comment by Mark9013100** [comment]
Is an issue on Bay as well.

**Comment by EmperorJon** [comment]
Cable/mouse traps don't work either. Looks like a surgery step issue not a hemostat issue.

**Comment by EmperorJon** [comment]
Okaaay so.
A while back some defines got moved around. This includes the creation of organ_damage.dm.
For whatever reason ORGAN_BLEEDING used to be 8 and is now 2. This means by chance that the surgery step for clamping bleeders evaluates to 2, and 2 seems to be the arbitrary return for "This step failed but the user has already been notified about it so just bail."
As a result when it attempts to evaluate the step it just bails out with no message or interaction of any sort.
In other words, I know what the /problem/ is but we'll need to look into what ORGAN_BLEEDING changing actually meant and what we should actually do to /fix/ it.

**Comment by EmperorJon** [comment]
Whatever, the failure notification is now -1 not 2. All seems to work fine.

**Comment by EmperorJon** [comment]
Fixed by #381.
